timemachine/ff/handlers/deserialize.py

```diff
@@ -20,8 +21,19 @@ def deserialize_handlers(obj):
 
     handlers = []
 
+    protein_ff = constants.DEFAULT_PROTEIN_FF
+    water_ff = constants.DEFAULT_WATER_FF
+
     for k, v in obj_dict.items():
 
+        if k == constants.PROTEIN_FF_TAG:
+            protein_ff = v
+            continue
```

Comment:
nit: is this equivalent to
```python
protein_ff = obj_dict.get(constants.PROTEIN_FF_TAG)
```
outside of the loop?

Reply:
yup, will fix

Reply:
Wouldn't this need to be `.pop` else it will fail on ln 52?